.gitignore

```diff
@@ -1,30 +1,401 @@
-﻿.project
-.idea/
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
 
-# Visual Studion tiedostoja
-*/[Ee]xported[Oo]bj/
-*.booproj
-*.pidb
+# User-specific files
+*.rsuser
 *.suo
-*.svd
 *.user
+*.userosscache
+*.sln.docstates
+
+# User-specific files (MonoDevelop/Xamarin Studio)
 *.userprefs
-bin/
-obj/
+
+# Mono auto generated files
+mono_crash.*
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x64/
+x86/
+[Ww][Ii][Nn]32/
+[Aa][Rr][Mm]/
+[Aa][Rr][Mm]64/
+bld/
+[Bb]in/
+[Oo]bj/
+[Ll]og/
+[Ll]ogs/
+
+# Visual Studio 2015/2017 cache/options directory
 .vs/
-Compiled/
-Compiled*/
-
-# Käyttiksen (Mac ja win) tekemiä tiedostoja joita jättää pois
-._*
-.DS_Store*
-.Spotlight-V100
-.Trashes
-ehthumbs.db
-Icon?
-[Tt]humbs.db
-
-# Dolphin
-.directory
-
-.idea/
```

Comment:
Don't remove these gitignore items that filter out the OS-specific garbage

Reply:
Items have been added